lib/src/widget/pin_widget.dart

```diff
@@ -52,8 +52,8 @@ class PinInputTextField extends StatefulWidget {
   /// Same as [TextField]'s enableInteractiveSelection
   final bool enableInteractiveSelection;
 
-  /// Same as [TextField]'s toolbarOptions
-  final ToolbarOptions? toolbarOptions;
+  /// Same as [TextField]'s contextMenuBuilder
+  final EditableTextContextMenuBuilder? contextMenuBuilder;
```

Comment:
this is a breaking change and broke [sms_autofill](https://pub.dev/packages/sms_autofill) package, I think it would be better to mark it as `@deprecated` and add the new `contextMenuBuilder` field

or when removing `toolbarOptions` bump it as a major version so 5.0.0

Reply:
I will check it and reproduce in my side